///////////////////////////////////// promises with async/await and try/catch method
// let promise = async () => {

//     try {
//         const data = await fetch('https://fakestoreapi.com/products/1')
//             .then(res => res.json());
//         console.log(data);
//     } catch (error) {
//         console.log(error.message);
//     }

// }
// promise()

///////////////////////////////////// modules in Java Script 




